docs/integrations/saas-cloud/docusign.md

```diff
@@ -0,0 +1,210 @@
+_sourceCategory="docusign_src"
+| json "object","userId","eventId","action","property","source","ipAddressLocation.latitude","ipAddressLocation.longitude","result","ipAddressLocation.city","ipAddressLocation.state","ipAddressLocation.country","data" as object,user_id,event_id,action,property,source,latitude,longitude,result,city,state,country,data nodrop
+| where object matches "{{object}}" and action matches"{{action}}" and source matches"{{source}}"
+| where country matches "{{country}}" or isNull(country)
+| where object matches("*Account*") or object matches("*Connect*") or object matches("*User*") or object matches("*PermissionSet*")
+| where action matches("*Updated*")
+| count_distinct(event_id) by _messageTime,object,action,property,data,source,result,city,state,country
+| formatDate(toLong(_messageTime), "dd-MM-yyyy HH:mm:ss") as time
+| if(isBlank(property),"nil",property) as property
+| top 100 time,object,action,property,data,source,result,city,state,country by time
+```
+
+```sql title="Alerts by Severity Over Time"
+_sourceCategory="docusign_src"
+| json "object","userId","eventId","action","property","source","ipAddressLocation.latitude","ipAddressLocation.longitude","result","ipAddressLocation.city","ipAddressLocation.state","ipAddressLocation.country" as object,user_id,event_id,action,property,source,latitude,longitude,result,city,state,country nodrop
+| where object matches "{{object}}" and action matches"{{action}}" and source matches"{{source}}"
+| where country matches "{{country}}" or isNull(country)
+| where object matches("*Alert*")
+| lookup alert,object,ds_action,description,severity,severity_id from https://sumologic-app-data.s3.amazonaws.com/docusign_alert_lookup.csv on action = ds_action
+| if(isNull(severity),"low",severity) as severity
+| where severity matches "{{severity}}"
+| timeslice 1d
+| count_distinct(event_id) as frequency by _timeslice,severity
+| fillmissing timeslice,values all in severity
+| transpose row _timeslice column severity as low, medium, high
+```
+
+## Installing the DocuSign App
+
+Locate and install the app from the **App Catalog**. To preview the dashboards included with the app before installing, click **Preview Dashboards**.
+
+Before you begin, collect logs from DocuSign API and ingest them into Sumo Logic. Refer to the [DocuSign API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/) to create the source and use the same source category while installing the app.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **DocuSign App** using the steps described in the [DocuSign Cloud-to-Cloud Source](docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/). If you already have set up your data, skip this step by clicking on **Next**.
```

Comment:
Path doesn't exist: /docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/)

Reply:
It looks like it's [in this PR](https://github.com/SumoLogic/sumologic-documentation/pull/1855/files). 

@parthiv-sumo @keyur-sumo, are these PRs to be merged together? When can we push them?

Reply:
@kimsauce @JainM6 : This PR needs to be merged after [DocuSign Source PR](https://github.com/SumoLogic/sumologic-documentation/pull/1855/files).